pkg/e2e/workloads/workloads.go

```diff
@@ -0,0 +1,57 @@
+package workloads
+
+import (
+	"context"
+	"io/fs"
+	"os"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/expect"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/common/label"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/e2e-framework/klient/decoder"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+)
+
+func init() {
+	alert.RegisterGinkgoAlert("Workloads", "SD-SREP", "@sd-qe", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
+}
+
+var _ = ginkgo.Describe("Workloads", ginkgo.Ordered, label.HyperShift, label.E2E, func() {
```

Comment:
```suggestion
var _ = ginkgo.Describe("[Suite: e2e] Workloads", ginkgo.Ordered, label.HyperShift, label.E2E, func() {
```

Just adding back the suite string, so these tests get picked as part of e2e run. Once we fully switch the configs over to label filters, we can remove these strings.

Reply:
updated!